graphs/graphs/main.py

```diff
@@ -107,7 +145,12 @@ def main(donation_url_template, base_dir='.'):
     logging.getLogger('bokeh').setLevel(logging.WARNING)
     
     delay = 60 * 1
-    timeout = 15 
+    timeout = 15
+    
+    shifts = [['Zeta Shift',   0, 6],
+              ['Alpha Flight', 6, 12],
+              ['Dawn Guard',  12, 18],
```

Comment:
These shifts are not in the correct order.

Reply:
That is embarrassing